index.html

```diff
@@ -259,9 +259,10 @@ <h2>Introduction</h2>
 An ecosystem where [=verifiable credentials=] and
 [=verifiable presentations=] are expected to be useful
         </li>
-        <li>
-The use cases and requirements that informed this specification.
-        </li>
+        <p>
+[[[VC-USE-CASES]]][[?VC-USE-CASES]] is also an excellent resource for the use
+cases and requirements that informed this specification.
+        </p>
       </ul>
```

Comment:
```suggestion
      </ul>
      <p>
The use cases and requirements that informed this specification may be found
in [[[VC-USE-CASES]]] [[?VC-USE-CASES]].
      </p>
```

Reply:
s/may/can/ since MAY is normative language and some people get confused when we use normative language in lower case.

Reply:
Is using normative words in a lower-case, non-normative way generally discouraged in spite of the explicit mention that normative language is always UPPERCASE? There are quite a few other avoidable examples of this in the document.

Reply:
> Is using normative words in a lower-case, non-normative way generally discouraged in spite of the explicit mention that normative language is always UPPERCASE?

Yes, because people either ignore or don't yet absorb the discussion of these words only having their [RFC 2119](https://datatracker.ietf.org/doc/html/rfc2119) (issued in March 1997) meaning when/if they're all-caps ([RFC 8174](https://datatracker.ietf.org/doc/html/rfc8174), issued in May 2017). This is not really surprising, as they clearly had their RFC 2119 meaning regardless of case for *20 years* before RFC 8174 clarification only 6 years ago.